exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -80,7 +80,8 @@ func NormalizeSpanName(tag string) string {
 		case buf.Len() == 0:
 			continue
 		// handle valid characters that can't start the string.
-		case unicode.IsDigit(c) || c == '.' || c == '-':
+		// '-' creates issues in the UI so we skip it
+		case unicode.IsDigit(c) || c == '.':
```

Comment:
We can probably add a tag with a digit or `.` in a test case to test this branch, now that we don't handle `-` here?

Reply:
👍 yup will add test